app/views/admin/suppliers/_layout.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1 class="govuk-heading-l">Suppliers</h1>
+
+
+<div style="display: flex; flex-direction: row">
```

Comment:
Let's move that to css file.

Reply:
Or use something like govuk grid row, and two columns with width half inside of it.